executor/runtime/docker/docker.go

```diff
@@ -322,6 +322,18 @@ func stableSecret() string {
 	}
 	return net.IP(ipBuf).String()
 }
+func maybeAddOptimisticDad(sysctl map[string]string) {
+	if unix.Access("/proc/sys/net/ipv6/conf/default/use_optimistic", 0) == nil {
+		sysctl["net.ipv6.conf.default.use_optimistic"] = "1"
+	} else {
+		log.Warning("Not enabling use_optimistic")
+	}
+	if unix.Access("/proc/sys/net/ipv6/conf/default/optimistic_dad", 0) == nil {
```

Comment:
It looks like we were doing `all` and `default` before, but this happens before the interface is up (right?) if so then we just need default, so that is fine.